bower.json

```diff
@@ -25,7 +25,8 @@
     "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
     "marked-element": "polymerelements/marked-element#^1.0.0",
-    "prism-element": "PolymerElements/prism-element#^1.0.0"
+    "prism-element": "PolymerElements/prism-element#^1.0.0",
```

Comment:
I think you also need to add `url-bar` to the bower main or something. Not sure if the docs will pick it up otherwise